### ISSUE TITLE
Use a file-based, persistent storage

### ISSUE THREAD

**Issue by CosmoMyzrailGorynych** [CosmoMyzrailGorynych/cosmic-everyday#1]
TL;DR: if a user does not open Cosmic Everyday for three months, its data is wiped.

Currently, CE uses PouchDB, that uses IndexedDB as its underlying driver, that utilizes built-in chromium storage. This storage is considered being ephemeral: though it may be used as a storage for a database, it is not meant for long-term use. As a result, if a database is not used for a considerable amount of time, it is wiped out. It seems that for CE, it is a period of three months.

So, either PouchDB's backend storage should be changed to a local, file-based, more native adapter, or another DB should be picked.

**Comment by Bandito11** [comment]
@CosmoMyzrailGorynych Have this been fixed? I can try to change it to the one I use.

**Comment by CosmoMyzrailGorynych** [comment]
@Bandito11 no, it wasn't. Feel free to send a pull request or make a fork.
